### DISCUSSION TITLE
Full usage sample?

### DISCUSSION THREAD

**Discussion by StefanOltmann** [crowdin/crowdin-api-client-java#190] [General]
In my project I got a "lang" folder with the files `messages.txt` (english original) and `messages_de.txt` which follow the Java properties format. I want to use crowdin to upload those files and download versions for other languages.

Please add documentation how to use the SDK to do this.

**Comment by andrii-bodnar** [comment]
Hi @StefanOltmann,

currently, we don't have such examples. You can check the following projects for the real-world examples:

- https://github.com/crowdin/crowdin-cli
- https://github.com/crowdin/android-studio-plugin
